beoordelingstool_overwrite_shapefiles_dialog.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+ BeoordelingstoolOverwriteShapefilesDialog
+                                 A QGIS plugin
+ The Beoordelingstool is a QGIS plugin for judging the quality of the manholes and pipes of a sewer.
+                             -------------------
+        begin                : 2017-04-26
+        git sha              : $Format:%H$
+        copyright            : (C) 2017 by Nelen & Schuurmans
+        email                : madeleine.vanwinkel@nelen-schuurmans.nl
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+import os
+
+from PyQt4 import QtGui, uic
+from PyQt4.QtCore import pyqtSignal
+
+FORM_CLASS, _ = uic.loadUiType(os.path.join(
+    os.path.dirname(__file__), 'beoordelingstool_overwrite_shapefiles_dialog.ui'))
+
+
+
+class BeoordelingstoolOverwriteShapefilesDialog(QtGui.QDialog, FORM_CLASS):
```

Comment:
Dit was toch niet nodig?
We willen de gebruiker niet toestaan om per ongeluk zijn eigen werk weg te gooien: hij moet de shapefiles maar in een andere directory opslaan.